pandas/tests/test_index.py

```diff
@@ -133,6 +133,14 @@ def test_str(self):
         self.assertTrue("'foo'" in str(idx))
         self.assertTrue(idx.__class__.__name__ in str(idx))
 
+    def test_repr_max_seq_item_setting(self):
```

Comment:
``.indices`` has much more to test (and is customizable per sub-class), while ``.create_index`` is a single plain vanilla one. For the generic tests using ``.indices`` provides more coverage.